CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing to HypLL
+
+Thank you for considering contributing to HypLL! We are always looking for new contributors to help us implement the constantly improving hyperbolic learning methodology and to maintain what is already here, so we are happy to have you!
+
+We are looking for all sorts of contributions such as
+<ul>
+  <li>Bugfixes</li>
+  <li>Documentation additions and improvements</li>
+  <li>New features</li>
+  <li>New tutorials</li>
+</ul>
+
+# Getting started
+### Git workflow
+We want any new changes to HypLL to be linked to GitHub issues. If you have something new in mind that is not yet mentioned in an issue, please create an issue detailing the intended change first. Once you have an issue in mind that you would like to contribute to, create a branch from the corresponding issue as detailed in https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-a-branch-for-an-issue. 
```

Comment:
I'm not sure this works for people who are not invited to the repository. I'd recommend to either add detailed docs on how to fork and create a branch ([as we do at Lightly](https://github.com/lightly-ai/lightly/blob/master/CONTRIBUTING.md#pull-requests), for example) or to just brush over it assuming people know (like [here](https://github.com/opengovernment/opengovernment/blob/master/CONTRIBUTING.md#submitting-changes)).

Reply:
Yes, you are right! I changed it to forking the repo instead and added links to github's explanations on how to fork a repo and how to make a pr from a fork.